### DISCUSSION TITLE
Github Copilot Enterprise semantic indexing repo problem

### DISCUSSION THREAD

**Discussion by seewjruk** [community/community#122924] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

We find that the semantic indexing function in Github Copilot Chat in Github.com is not normal. All the repo that has been indexed before, including repos with only 1000 lines of code, it starts to index again and it has been keeping indexing for hours. 
According to the description of Github Copilot Chat in Github.com, the first indexing will take about half an hour while the update only takes about 5 minutes. But now it has been indexing for 3 hours, on a repo with only 1000 lines! 
<img width="436" alt="2024-05-08 154517" src="https://github.com/community/community/assets/156218583/cab0d675-9d12-4149-8623-e8873b982c5a">

**Comment by seewjruk** [comment]
Hi, is there anyone who can have a look at my issue? I have also submit support ticket, a service staff replied to me and said he would escalate the issue to tech team and there will be a response within one business day. But now there is no response and the problem still exists. ticket number is #2774848

**Comment by Adam-Abdulhalim** [comment]
We recently got Copilot Enterprise as well and are experiencing the same issue. Began indexing yesterday morning and it still shows "Copilot is indexing this repository. This may take a few minutes." an entire day later. We are thinking it may be due to the repo being very active with changes. Wish we had a definitive reason as to why this is happening.

**Reply by seewjruk** [reply]
Finally I see someone who has the same issue! I don't think it's about the activity of your repo. I have created a new repo with only one readme.md in the repo and write two words in it, the copilot has been indexing this repo for a whole day!

**Reply by Adam-Abdulhalim** [reply]
Wow, there must be something wrong with it then. Please keep us updated here if you can if you get a solution/response! I will do the same. Would love to know what the issue is so we can start utilizing this.

**Reply by seewjruk** [reply]
I have tried many methods, including shutting down and reopen the Github Copilot Enterprise, moving the enterprise owner to another account, enabling all the options of copilot policy, and I even started a new enterprise account to see if this problem exists, and all these measures are invalid, it cannot index even a single line of code in our repo. 

It is also just weird that the previously indexed repos suddenly lose all the semantic indexing even though I didn't update them at all. 

Some very hot public repos show the green words of "has been indexed", this is even more weird since they don't belong to our account at all.

Nobody from Github recognize this problem now, it just seems that the 20 more dollars per sear paid than the business version is just useless and meaningless.

**Reply by Adam-Abdulhalim** [reply]
So odd. As of the last hour or so it seems our screen has changed from what you posted above to now this:

![Screenshot 2024-05-09 123623](https://github.com/community/community/assets/119151435/3f4f5a39-832d-49d6-b25b-a84eb166e638)

So maybe it's actually making some progress?

**Reply by Adam-Abdulhalim** [reply]
![image](https://github.com/community/community/assets/119151435/f85931a9-4513-40dd-bc2b-13248fcf68a9)

Seems its working now, thankfully. Took almost 48 hours for it to finish indexing for whatever reason, very weird.

**Reply by seewjruk** [reply, marked as answer]
The have developed a hotfix for this issue, I submitted the ticket and they find out a way to solve this problem.
